src/tests/ftest/container/container_async.py

```diff
@@ -134,7 +134,7 @@ def test_destroyasync(self):
         """
         Test container destroy for asynchronous mode.
 
-        :avocado: tags=container,containerasync,contdestroyasync
+        :avocado: tags=all,small,full_regression,container,containerasync,contdestroyasync
```

Comment:
It seems that renaming `full_regression` to something shorter like `full`, `reg`, or `fr` would solve these pylint-line-too-long errors.  Is that an option?

Reply:
Yeah arguably something shorter would be better but being longer guarantees nobody will accidentally reuse it as might happen with something shorter.  Also, I don't want to edit every file again.